src/solvers/dgsem_p4est/dg_2d.jl

```diff
@@ -20,32 +20,68 @@ function create_cache(mesh::P4estMesh{2}, equations, mortar_l2::LobattoLegendreM
 end
 
 
+# TODO: p4est interface performance, move and generalzie this function for 3D
+@inline function index_to_start_step(index::Symbol, idx_one::Int, idx_end::Int)
+  if index === :one
```

Comment:
I suggest moving this to an issue since I didn't introduce this convention. Changing it here means that I also need to change the 3D stuff, which I don't want to do right now. This question is completely orthogonal to the purpose of this PR.

Reply:
:+1:

Reply:
Done in #777

Reply:
Thanks!